src/nnet3/nnet-am-decodable-simple.h

```diff
@@ -392,6 +392,170 @@ class DecodableAmNnetSimpleParallel: public DecodableInterface {
 };
 
 
+/*
+  This class handles the neural net computation in parallel.
```

Comment:
I think you should be clearer about this since the code itself is no more parallel than the normal code.
Maybe something like:
 ```
This class does neural net inference in a way that is optimized for GPU use: it combines
chunks of multiple utterances into minibatches for more efficient computation. 
```